models/seaport/ethereum/seaport_ethereum_base_pairs.sql

```diff
@@ -1,6 +1,15 @@
 {{ config(
-     alias = 'base_pairs'
-     )
+    alias = 'base_pairs',
+    partition_by = ['block_time'],
+    materialized = 'incremental',
```

Comment:
This shouldn't be incremental. There's no incremental logic in the section below. Can you turn this into a table instead?

Reply:
I pasted this from other models like `seaport_optimism_base_pairs`. I honestly don't know: what would be better? To have this be a table, or to have this be an incremental table (and add the if incremental jinja bits to the model)?

Reply:
:warning:  This *should* be made incremental, or it will make the seaport.trades model only update once a day.

Reply:
Okay, let me fix that 👍 thanks!